jsonschema-core.xml

```diff
@@ -350,9 +350,9 @@
                         and its companions, are free to define other behaviors as well.
                     </t>
                     <t>
-                        A JSON Schema MAY contain properties which are not schema keywords.
-                        Unknown keywords SHOULD be treated as annotations, where the value
-                        of the keyword is the value of the annotation.
+                        A JSON Schema MAY contain properties which are not schema keywords or are not recognized as schema keywords.
```

Comment:
I think it'd suffice to say "A JSON Schema MAY contain properties which are not recognized as schema keywords."

Reply:
oh, I see this was the subject of the resolved review. no need to relitigate that if it's been sufficiently addressed, disregard if so.

Reply:
@notEthan I preserved the "not schema keywords" part from the original text because I don't remember why it was phrased that way and did not want to get into a debate about whether it is OK to remove it.  If you think it should be removed, we should probably discuss that in an issue.  I figured adding "or are not recognized as..." would cover the more common way this is thought about without removing anything previously allowed.

Reply:
sounds good to me